src/main/Port.Adapter/UI/Views/Blazor.Common/TreeView.razor

```diff
@@ -89,10 +90,10 @@
                                         }
                                     }
                                 </div>
-                                <div onmouseover="hover('g', this.id, true, highlightNode)" onmouseout="hover('g', this.id, false, highlightNode)" class="col-auto p-1" 
-                                    id="@child.Neuron.Id" style="@(this.GetStyleByCreationAuthor(child.Neuron.Validation.IsCurrentUserCreationAuthor) + 
+                                <div @onmouseover="async () => await Hover(child)" @onmouseout="async () => await UnHover(child)" class="col-auto p-1"
```

Comment:
Fix capitalization. "Un" is not a separate word therefore it should be "Unhover."

Reply:
Done